src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -508,14 +513,31 @@ class SignUpModalController {
     // The loading icon will be shown, so the user will know something is happening.
     const mfaOptions = document.querySelector('.select-authenticator-enroll')
     if (mfaOptions) {
+      // Hide the options immediately and wait 10 seconds or for the user to click continue
       mfaOptions.style.display = 'none'
-      // As Okta enforces the MFA screen to be shown during sign up, we need to skip it to streamline the process.
-      // The user can setup MFA at a later date in their account settings.
-      const skipMFAButton = mfaOptions.querySelector('.button.skip-all')
-      skipMFAButton?.click()
+
+      this.oktaRedirectCountdown = 10
+      this.oktaRedirectInterval = this.$interval(() => {
+        --this.oktaRedirectCountdown
+        if (this.oktaRedirectCountdown === 0) {
+          this.completeRegistration()
+        }
+      }, 1000, 10)
```

Comment:
The third argument `10` here makes the interval automatically cancel after 10 times.

https://docs.angularjs.org/api/ngMock/service/$interval#$interval-arguments